mobly/controllers/android_device_lib/adb.py

```diff
@@ -29,8 +29,8 @@
 
 # Number of attempts to execute "adb root", and seconds for interval time of
 # this commands.
-ADB_ROOT_RETRY_ATTMEPTS = 3
-ADB_ROOT_RETRY_ATTEMPT_INTERVAL_SEC = 10
+ADB_ROOT_RETRY_ATTEMPTS = 3
+ADB_ROOT_RETRY_ATTEMPT_INTERVAL_SEC = 15
```

Comment:
I'd keep this as the old value, but increase by multiplying by 1.5 or 2 instead of adding 2

Reply:
I do multiply by 2 indeed see line 535.
But keeping it to 10 wouldn't be enough, as it will be in total 30 seconds, and sometimes (although rarely) it took few seconds more)

Reply:
Oh sorry, I didn't read it carefully.

I'm a little hesitant to raise the baseline though.

It's 3 retries right?
So the total can be well above 30s?

Reply:
It's 3 attempts but the first one counts as attempt, so it's
root - wait 10 root - wait 20 - root

Reply:
I retried with 10 seconds as baseline, repeated 20 times on the same device and passed all the times. If you prefer not to increase the baseline I think that 10s with multiplier of 2 already prevents most of the timeouts issues. I will fix this and the CLA warning tomorrow

Reply:
sg, thx